src/facade/memcache_parser.cc

```diff
@@ -38,6 +38,15 @@ MP::CmdType From(string_view token) {
 
 using TokensView = absl::Span<std::string_view>;
 
+bool TokenToExpireTs(std::string_view token, std::uint32_t* expire_ts) {
+  static constexpr uint32_t month = 60 * 60 * 24 * 30;
```

Comment:
pls remove  static and rename to `kExpireLimit`. The limit is according to their documentation?

Reply:
Yes see https://www.php.net/manual/en/memcached.expiration.php

Maybe it's also worth adding a comment :)

Reply:
definitely worth adding the link and the comment. I still think we should not add time_now inside parser.

Reply:
I agree, also I wanted to add the same comment (to include the link) but somehow I messed this up :(